lib/utils/auth.js

```diff
@@ -51,7 +51,7 @@ const adduser = async (npm, { creds, ...opts }) => {
   if (!res) {
     const username = await read.username('Username:', creds.username)
     const password = await read.password('Password:', creds.password)
-    const email = await read.email('Email: (this IS public) ', creds.email)
+    const email = await read.email('Email (this will be public):', creds.email)
```

Comment:
this is an unrelated `docs:` change that we don't want to hide in this PR under the `fix:`